doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -0,0 +1,373 @@
+# The Node.js Event Loop, Timers, and `process.nextTick()`
+
+## What is the Event Loop?
+
+The event loop is what allows Node.js to perform non-blocking I/O
+operations — despite the fact that JavaScript is single-threaded — by
+offloading operations to the system kernel whenever possible. 
+
+Since most modern kernels are multi-threaded, they can handle
+multiple operations executing in the background. When one of these
+operations completes, the kernel tells Node.js so that the appropriate callback 
+may added to the `poll` queue to eventually be executed. We'll explain
+this in further detail later in this topic. 
+
+## Event Loop Explained 
+
+When Node.js starts, it initializes the event loop, processes the provided
+input script (or drops into the REPL, which is not covered in this document)
+which may make async API calls, schedule timers, or call `process.nextTick()`, 
+then begins processing the event loop. 
+
+The following diagram shows a simplified overview of the event loop's
+order of operations.
+
+       ┌───────────────────────┐
```

Comment:
@piscisaureus 
> close callbacks (e.g socket.on('close', ...))

That's not always true for some odd reason. The only time the `'close'` event is emitted in `uv__run_closing_handles()` is on `Socket#_destroy()`. Otherwise it simply runs in `nextTick()` (example `Server#_emitCloseIfDrained()` and subsequent call to `emitCloseNT()`).